File: Python/1252. Cells with Odd Values in a Matrix.py
class Solution:
    def oddCells(self, m: int, n: int, indices: List[List[int]]) -> int:
        x, y = [0] * m, [0] * n

        for i, j in indices:
            x[i] += 1
            y[j] += 1

        return len([1 for i in x for j in y if (i + j) % 2 == 1])

File: Java/1252. Cells with Odd Values in a Matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int x[] = new int[m], y[] = new int[n], ans = 0;

        for(int[] i: indices){
            x[i[0]] += 1;
            y[i[1]] += 1;
        }

        for(int i: x){
            for(int j: y){
                if((i + j) % 2 == 1) ans++;
            }
        }

        return ans;
    }
}
